### ISSUE TITLE
Do not block Spawn Ratio 0.0 value 

### ISSUE THREAD

**Issue by ve1oxby** [WerewolfCustoms/ModsStudio2IssueBoard#178]
**Describe the bug**
UI is blocking 0.0 value for Spawn Ratio
This is allowed by SCS and could be used to some mods ([ref](https://modding.scssoft.com/wiki/Games/ETS2/Modding_guides/1.35#Trailer_chains)) . It is better to make a notification window instead of hard blocking the value. 

**Screenshots**
![image](https://user-images.githubusercontent.com/51075160/153276353-245a8bee-3fbd-4512-859d-3a609c8a7c83.png)

**Desktop (please complete the following information):**
 - Version v.2021.03

**Comment by pete-agreatguy** [comment]
I'll start looking into this next week.

I remember we devs having an in depth discussion about the values we should allow. 0 was dismissed because of something major which eludes me at this moment.

@ve1oxby Is this value of 0, wanted only on the traffic trailer value or other ai vehicle values also? If yes, which?

Thank you for your report and using the correct method of reporting :-)

**Comment by pete-agreatguy** [comment]
Right; now that I am more awake (it was 11pm when I replied last night my time), I can provide a better answer.

First of all; the reference you are linking to was for game changes in v1.35. There have been many changes since then :)

That way no longer works for combos sadly. Since game version 1.38 (I think off the top of my head), combos have been achieved using trailer chains.

If you test changing the spawn value for the trailer to 0 (def\vehicle\trailer\[trailer def file]), you will find combos do not spawn in game. However, if set to 0.1 then combos do appear (including the removal of trailer_semi of course). This is the only way to get combos to work correctly. Granted, there is a very slim chance the trailer will spawn with another truck but it is extremely unlikely that a player will see this. Heck, when testing traffic set to 9.9, I am yet to see a trailer spawn with another truck other than the combo truck I have defined in the trailer chain (combo) :)

Notes: 
trailer_semi is a unit used in SCS games to represent all AI trailers.
truck_semi is used to identify single trailers; it has no other use

The only thing that has changed recently is the use of how to get certain chains to spawn in certain areas (this means I need to update my tutorial at some point on discord). Saying that, SCS haven't disabled the country def files way so it will still work.

Going to close this as it is working as intended at the moment until SCS make further enhancements in the traffic ai system.

PS: The above is how jazzycat also gets combos to spawn. Feel free to check out his def files :)
